code/script.js

```diff
@@ -1,11 +1,118 @@
 // Variables that point to selected DOM elements
 const chat = document.getElementById('chat');
+const inputWrapper = document.getElementById('input-wrapper');
 
 // If you need any global variables that you can use across different functions, declare them here:
-
+let question = 0;
 
 // Declare your functions after this comment
 
+  const getEmail = () => {
+    showMessage('Enter your email below', 'bot');
+    inputWrapper.innerHTML = 
+    `
+    <form id="name-form">
+          <label for="name-input"></label>
+          <input id="name-input" type="email" />
+          <button class="send-btn" type="submit">
+            Send
+          </button>
+    </form>
+    `
+    question++;
+
+    document.getElementById('name-form').addEventListener('submit', (event) => {
+      event.preventDefault()
+      const nameInput = document.getElementById('name-input');
+      const email = nameInput.value
+      setTimeout(() => showMessage (`${email}`, 'user'))
+      setTimeout(() => showMessage(`Your appointment is booked. We will send you a confirmation at <em>${email}</em>`, 'bot'), 1000) ;
+      setTimeout(() => nextQuestion(), 2000);
+      nameInput.value = ''
+  
+  })
+  }
+
+  const showEmergencies = () => {
+    inputWrapper.innerHTML = `
+    <button id="bleached">Help! I accidently bleached my hair</button>
+    <button id="bald"> I woke up bald!</button>
+    <button id="wedding">Need a last minute wedding-do!</button>`
+  
+    document.getElementById("bleached")
+    .addEventListener('click', () => setTimeout(() => showBleachedOptions(), 1000));  
+    document.getElementById("bald")
+    .addEventListener('click', () => setTimeout(() => showBaldOptions(), 1000)); 
+    document.getElementById("wedding")
+    .addEventListener('click', () => setTimeout(() => showWeddingOptions(), 1000)); 
+  }
+
+  const showBleachedOptions = () => {
+    showMessage("Accidentally bleached my hair!", 'user');
+    setTimeout(()=> showMessage("Oh no! 😱 What can we do for you?", 'bot'), 2000);
+    inputWrapper.innerHTML = `
+    <select id="select" class="select">
+        <option value="" selected disabled>Select your fix</option>
+        <option value="Color it back to normal">Color it back to normal</option>
+        <option value="Cut it off">Cut it off</option>
+    </select>
+    `
+    question++; 
+    document.getElementById('select').addEventListener('change', () => setTimeout(() => showMessage(select.value, 'user'), nextQuestion()));
+  }
+
+  const showBaldOptions = () => {
+    showMessage("I woke up bald!", 'user');
+    const wig = `<a href=https://www.wig.se/>wig.se</a>`
+    setTimeout(showMessage(`Sorry we cant help you 🤷‍♀️. Perhaps you can get help at ${wig}`, 'bot'), 2000);
```

Comment:
For all buttons there is a delay of 2000 milliseconds, which is very nice, but it seams that it has fallen out in this section because the answer from the bot appears at the same time as user.